source/crd.go

```diff
@@ -118,14 +118,35 @@ func (cs *crdSource) Endpoints() ([]*endpoint.Endpoint, error) {
 
 	for _, dnsEndpoint := range result.Items {
 		// Make sure that all endpoints have targets for A or CNAME type
-		for _, endpoint := range dnsEndpoint.Spec.Endpoints {
-			if (endpoint.RecordType == "CNAME" || endpoint.RecordType == "A") && len(endpoint.Targets) < 1 {
-				log.Warnf("Endpoint %s with DNSName %s has an empty list of targets", dnsEndpoint.ObjectMeta.Name, endpoint.DNSName)
+		crdEndpoints := []*endpoint.Endpoint{}
+		for _, ep := range dnsEndpoint.Spec.Endpoints {
+			if (ep.RecordType == "CNAME" || ep.RecordType == "A") && len(ep.Targets) < 1 {
```

Comment:
Some providers already support AAAA, I think we also should check them just in case

Reply:
Added them.